src/components/ConditionalFilter/API/InitialStateResponse.ts

```diff
@@ -31,6 +31,10 @@ export class InitialStateResponse implements InitialState {
     return this.attribute[name];
   }
 
+  public static empty() {
+    return new InitialStateResponse([], {}, [], [], []);
+  }
+
```

Comment:
**suggestion**: InitialStateResponse has 5 parameters - this is unreadable, please use object parameter.